docs/running-on-yarn.md

```diff
@@ -495,6 +468,20 @@ To use a custom metrics.properties for the application master and executors, upd
   name matches both the include and the exclude pattern, this file will be excluded eventually.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.report.interval</code></td>
```

Comment:
If it wasn't documented, it was on purpose.  If users find it a useful config we can expose it.